packages/drivers/routerlicious-driver/src/cache.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { MapWithExpiration } from "@fluidframework/driver-utils";
+import { MapWithExpiration } from "./mapWithExpiration";
```

Comment:
Is the intention to fully remove MapWithExpiration or just not export it at this point?

Reply:
Just to not export it.  I believe we have many cache-like behaviors in the repo that we could consider consolidating at some point, but probably not a priority.